CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Pulumiverse
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing. These are just
+guidelines, not rules, so use your best judgment and feel free to propose
+changes to this document in a pull request.
+
+## Issues and Pull Requests
+
+- If you're not sure about adding something, open an issue to discuss it.
+- Feel free to open a Pull Request early so that a discussion can be had as
+  changes are developed.
```

Comment:
Add:

> Mark it as `draft` if you consider your contribution not yet complete.

Reply:
I'm not sure we should have this mentioned two times; IMHO, it isn't super important. I wouldn't care if people opened pull requests that are not yet complete.